src/main.cpp

```diff
@@ -2170,13 +2269,27 @@ void explode(int x, int y, int z) {
     }
 }
 
-// Destroy block at given x, y, z.
-void destroyBlock(int x, int y, int z) {
-    set_block(x, y, z, 0);
-    record_block(x, y, z, 0);
+// Destroy Block
+extern "C"
```

Comment:
Why did you swap the function placement? I don't mind, but I want to know if I was making a mistake with how I had them before.

Reply:
I don't recall. It may have been a C++ function called before definition type of thing.

Reply:
It probably was. I went ahead and fixed it by adding a function declaration. Kind of sloppy of me, but since I'm planning to extract the TNT functions to their own files soon anyway.